### ISSUE TITLE
Make DOM order of Scrollable and Dropdown navigation consistent between canonical and AMP

### ISSUE THREAD

**Issue by tochwill** [bbc/simorgh#5301]
**Describe the bug**
In the canonical navigation, we render Scrollable then Dropdown navigation in the DOM. In the AMP navigation, we render Dropdown and then Scrollable. Only one is visible at a time, but it would be nice if this was consistent.

**Expected behaviour**
The navigations should be rendered in the same order.

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Testing notes**
- Check that this doesn't cause any regressions for any items on the accessibility checklist: https://docs.google.com/spreadsheets/d/1jti0QNgnMYNOY9L0qQEdcciuVBWkZjYjF6L0NZaX9RE/edit#gid=0
- [ ] This bug fix is expected to need manual testing.

**Additional context**

**Comment by tochwill** [comment]
Blocked on https://github.com/bbc/simorgh/pull/5234

**Comment by greenc05** [comment]
And it would be nice if the DOM order was dropdown, then scrollable, as this makes most sense in the mark up order e.g. as per AMP

**Comment by aeroplanejane** [comment]
Closing as won't do. cc @gavinspence
